quay-tectonic-identity.md

```diff
@@ -0,0 +1,72 @@
+# Quay Enterprise OIDC Auth with Tectonic Identity 
+
+## Export certificate information from cluster for TLS verification of Tectonic Identity: 
+
+```
+kubectl -n tectonic-system get secret tectonic-ca-cert-secret -o json | jq -r '.data[]' | base64 -d >> tectonic-ca-ingress.crt
+kubectl -n tectonic-system get secret tectonic-ingress-tls-secret -o json | jq -r '.data["tls.crt"]' | base64 -d >> tectonic-ca-ingress.crt
+```
+## Upload Cert to QE
+
+Upload the `tectonic-ca-ingress.crt` certificate to QE via the superuser panel or follow the [documentation](https://coreos.com/quay-enterprise/docs/latest/insert-custom-cert.html) upload manually. 
```

Comment:
... to Quay through the superuser panel, or follow [Adding TLS Certificates to the Quay Enterprise Container][add-certs] to upload manually.